### PR TITLE
improve help and validation messages

### PR DESCRIPTION
**Pull request by azazellochg**
image
image
Also set default mag=50000

### CODE REVIEW

**Review by JorMaister — APPROVED** [review]

**Comment by JorMaister** [comment]
That's nice. I like it! Many thanks, @azazellochg

File: tomo/convert/mdoc.py [hunk 1]
```diff
@@ -224,8 +224,8 @@ def _getSlicesData(self, zSlices, tsFile):
         if round(accumulatedDose) > 0:
             self.mdocHasDose = True
         else:
-            logger.warning("Dose not found or almost 0 (%s) in %s" % (accumulatedDose, self._mdocFileName) )
-
+            logger.debug("Dose not found or almost 0 (%s) in %s" %
```
Comment (pconesa) [inline tomo/convert/mdoc.py:227]: Isn't this information useful to the user by default?

Comment (azazellochg) [reply tomo/convert/mdoc.py:227]: it's printed in the warnings already (see picture above), I dont know what's the purpose of this extra message.

Comment (pconesa) [reply tomo/convert/mdoc.py:227]: If we allow to proceed without dose, it would be nice to have it in the log just in case.

Comment (azazellochg) [reply tomo/convert/mdoc.py:227]: I've simulated a test where the first mdoc has dose and the second doesn't. The protocol runs and I have in stdout:
```
00037:   The following mdoc files were skipped:
00038:       /home/gsharov/test/imod/dose_test/TS_03.mrc.mdoc
00039:   
00040:   /home/gsharov/test/imod/dose_test/TS_03.mrc.mdoc is missing:
00041:     - Dose values. Related mdoc labels are: ExposureDose or FrameDosesAndNumber or (DoseRate and ExposureTime) or (MinMaxMean and CountsPerElectron)
```
I'm not sure why you want to print the same dose warning again...

**Comment by pconesa** [comment]
As long as it is in the log I'm ok.

**Pull request merged**